modules/dxp/apps/portal-search-elasticsearch-cross-cluster-replication/portal-search-elasticsearch-cross-cluster-replication-impl/src/main/resources/content/Language.properties

```diff
@@ -1,7 +1,7 @@
 cross-cluster-replication-configuration-name=Cross-Cluster Replication
 cross-cluster-replication-enabled=Enabled
-cross-cluster-replication-enabled-help=Enable or disable read operations from the local cluster. This setting will only be used when Operation Mode is set to REMOTE. The Operation Mode is defined in the Elasticsearch System Settings.
-cross-cluster-replication-local-cluster-connection-id=Cross-Cluster Replication Local Cluster Connection ID
-cross-cluster-replication-local-cluster-connection-id-help=Set the connection ID for the local cluster that will be used for read operations. Connections are defined in the Elasticsearch Connections System Settings.
+cross-cluster-replication-enabled-help=Enable or disable read operations from the local Elasticsearch clusters. This setting will only be used when Operation Mode is set to REMOTE. The Operation Mode is defined in the Elasticsearch System Settings.
```

Comment:
@rbohl if you have a min to look at these changes let me know if you have any suggestions. not sure if we should still call it a portal, or maybe DXP since its CCR stuff... etc
im done for the day and ill be on PTO monday, so no rush.
also fyi, ill have some screenshots and updated configs/setup for the changes introduced in this LPS

Reply:
> cross-cluster-replication-enabled-help=Enable or disable read operations from the local Elasticsearch cluster. This setting is only used when Operation Mode is set to REMOTE. Configure the Operation Mode in the Elasticsearch System Settings entry.

That's just a little wordsmithing, but I did change "clusters" (plural) to "cluster" because it seemed more accurate to me. Feel free to change it back to plural if I misunderstand.
__________________
>  cross-cluster-replication-local-cluster-connection-ids=Cross-Cluster Replication Local Cluster Connection IDs

This sounds arcane, but I don't know if we can do anything about it.  If we renamed to "Read-Only (Local) Connection IDs" would it be less accurate or clear in your opinion?
______________
>  cross-cluster-replication-local-cluster-connection-ids-help=Use key=value format to map each portal address to a read-only (local) Elasticsearch cluster's connection ID. Follow the pattern host:port=connectionId. Connections are defined in the Elasticsearch Connections System Settings.

Trying to figure this one out, hopefully it's still accurate. I assume there can be multiple values? If so we should add that they're comma-separated or whatever the required format is.
_____________
Let me know if you think I made these worse, haha.